### PR TITLE
[exporter/loki] remove opencensus usage

### LINKED ISSUE #30170: exporter/loki

### PR DESCRIPTION
**Pull request by codeboten**
Fixes #30170

### CODE REVIEW

File: exporter/lokiexporter/exporter.go
Comment (bogdandrutu) [inline exporter/lokiexporter/exporter.go:50]: I see this more and more, should we actually generate an "obsrep" equivalent from metadatagen? I will give it a try maybe during the break.

File: exporter/lokiexporter/exporter.go
Comment (bogdandrutu) [inline exporter/lokiexporter/exporter.go:46]: I would not ignore the error if possible (a.k.a not too many changes to return error in newExporter).

**Comment by codeboten** [comment]
Pinging code owners for review: @gramidt @gouthamve @jpkrohling @mar4uk

**Comment by mar4uk** [comment]
LGTM!

File: exporter/lokiexporter/exporter.go
Comment (codeboten) [reply exporter/lokiexporter/exporter.go:46]: added error handling

**Comment by codeboten** [comment]
@mar4uk just updated the PR to add error handling, please approve if the changes look good

**Review by mar4uk — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**
